packages/datadog-instrumentations/src/pug.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const shimmer = require('../../datadog-shimmer')
+const { channel, addHook } = require('./helpers/instrument')
+
+const pugReadCh = channel('datadog:pug:compile:start')
```

Comment:
**Nit**: Var name could be improved by correctly indicating the operation to be published in the DC. In this case, as it will publish a message every time pug compiles, it could be named `pugCompileCh`.

Reply:
Done